ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -175,32 +170,32 @@ spec:
 EOF
 ```
 
-This creates a new configuration with node-to-node mesh BGP disabled.
-If you have a configuration, and `meshenabled` is set to `true`, update your
-configuration:
+This action creates a new configuration with node-to-node mesh BGP disabled.
 
-```
-calicoctl get bgpconfig --output yaml > bgp.yaml
-```
+If you have a configuration, and `meshenabled` is set to `true`:
 
-Edit the `bgp.yaml` file, updating `nodeToNodeMeshEnabled` to `false`. Then
-update Calico configuration by running:
+1. Update your configuration by running:
 
-```
-calicoctl replace -f - < bgp.yaml
-```
+    ```
+    calicoctl get bgpconfig --output yaml > bgp.yaml
+    ```
+
+2. Edit the `bgp.yaml` file to change `nodeToNodeMeshEnabled` to `false`. 
+3. Update the Calico configuration by running:
+
+    ```
+    calicoctl replace -f - < bgp.yaml
+    ```
 
-## Configure Calico to use Route Reflectors
+## Configure Calico to use route reflectors
 
-To configure Calico to use the Route Reflectors you need to know the AS number
-for your network first. For that, run:
+To configure Calico to use route reflectors, you first need to know the autonomous system (AS) number for your network. To get the AS number for your network, run:
 
```

Comment:
I suggestion changing the sentence to read: "To configure Calico to use route reflectors, you first need to get the Autonomous System (AS) number for the network."